resources/views/teams/details.blade.php

```diff
@@ -0,0 +1,34 @@
+
+@extends('layouts.app')
+
+@if(isset($team))
+	@section('title', $team['name'])
+
+	@section('content')
+	<div class="padded content">
+		<h1>{{ $team['name'] }}</h1>
+		<div class="team-info">
+			<div class="image-container">
+				<img id="avatar" src="{{ $team['photo'] }}" alt="Team avatar" class="ui small image"/>
```

Comment:
En algunos casos usas `<img ...>` y en otros `<img />`. Let's try to be consistent